src/corehost/cli/nethost/nethost.h

```diff
@@ -35,6 +35,23 @@
 extern "C" {
 #endif
 
+// Parameters for get_hostfxr_path
+//
+// Fields:
+//   assembly_path
+//     Path to the compenent's assembly.
+//     If specified, hostfxr is located as if the assembly_path is the apphost
+//
+//   dotnet_root
+//     Path to directory containing the dotnet executable.
+//     If specified, hostfxr is located under this path and assembly_path is ignored.
+//
+struct get_hostfxr_parameters {
+    size_t version;
```

Comment:
What should the value of `version` be? I would instead suggest the size of the struct - which is a Windows-ism, but does hold up well.

Reply:
Oops, missed documenting. Yes, it should be the size of the struct.